tools/java/org/unicode/cldr/util/data/subtypeMapping.txt

```diff
@@ -0,0 +1,28 @@
+# -*- BEGIN CheckCLDR.Subtype Mapping -*-
+# Format is URL "http..." followed by Subtype values, the latter separated by commas.
+# Each of the Subtypes is covered by documentation at that URL.
+# Comment lines must START with a #, because # is used for HTML fragments.
+
+
+
+http://cldr.unicode.org/translation/timezones#metazones
```

Comment:
Should be https -- but I see that currently fails for cldr.unicode.org. Can the distinction be encapsulated in a single place?

Reply:
I think for editing purposes it would be easiest to have full URLs. Doesn't look like we aren't going to get https on google sites.  Since it's plain text, I think we can do a bulk rename when/if we move to something else.

So, I would like to keep http://cldr.unicode.org for now, pending the overall discussion about moving the content somewhere else.  The goal of this ticket is to be an improvement of what we have and decouple the URLs from the source code.